Marlin/Configuration_adv.h

```diff
@@ -2980,7 +2980,7 @@
    *  - RPM     (S0 - S50000)  Best for use with a spindle
    *  - SERVO   (S0 - S180)
    */
-  #define CUTTER_POWER_UNIT PWM255
+  #define CUTTER_POWER_UNIT PERCENT
```

Comment:
The display will be in %, there is no issue keeping it at PWM. I would think anyone who can compile and upgrading should be aware of what mode they require for M3 commands.

Reply:
I am thinking about the way I upgrade my configurations. I rebase my configurations on the latest `bugfix`. If a previous default was good enough for me, it will not cause a merge conflict if it changes. I will only see a conflict if I had actually changed the value.

Reply:
FYI, Smoothie firmware defaults to %, GRBL default is PWM so it's a toss up. Maybe we should keep it as PWM for safety then.

Reply:
That is probably the safest things to do. To double-check, the display is always in percent, no matter that this is set to?

Reply:
Ok, I set it to PWM. The display is dynamic, if it set at PWM you get 0-255.

Reply:
There are edits outside of my testing, I see the issue, the test for PWM or PERCENT is nested and hiding it. going to run it through VS and then update it.

Reply:
You refer to `CUTTER_UNIT_IS`?

I found that when I was exploring, trying to understand if this changed only the display, or also G-Code processing.

Reply:
The display is what it changed, however it pulls it from the existing code.

Reply:
Since Marlin is based off of GRBL, we want to keep the same default. I am not oppose to having the LCD power display in a different unit, however I think it'll just add to the confusion. There is also the possible rounding error.